projection/ibc_channel/view/ibc_channels.go

```diff
@@ -460,6 +544,10 @@ func (ibcChannelsView *IBCChannels) List(
 		"view_ibc_channels",
 	)
 
+	if chainID != nil {
```

Comment:
is it better to have a separated func commonListIBCChannelsByChianId?

Reply:
Sure! I did not do that mainly due to not wanting to duplicate code...

But yes, if we do so, it will be easier to maintain and not easy to break thing when we change the code. Let me update the code then!